### PR TITLE
fix some things and add working adapter testing

### PR DESCRIPTION
**Pull request by Apollon77**
Have fun :-)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,12 +39,12 @@
     "nyc": "^11.0.2",
     "typescript": "^2.4.2"
   },
-  "main": "build/main.js",
+  "main": "main.js",
```
Comment (AlCalzone) [inline package.json]: how does ioBroker know to look for the main.js file in the build directory?

**Comment by AlCalzone** [comment]
Also please update `/src/lib/utils.ts`, lines 8..12. All occurrences of 3 should be a 4.

**Comment by Apollon77** [comment]
Thanks for feedback, changed and tests running again!!

**Comment by AlCalzone** [comment]
Thanks!

**Pull request merged**